src/main/java/org/apache/sling/models/jacksonexporter/impl/JacksonExporter.java

```diff
@@ -22,17 +22,16 @@
 import java.io.StringWriter;
 import java.util.Map;
 
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Reference;
-import org.apache.felix.scr.annotations.ReferenceCardinality;
-import org.apache.felix.scr.annotations.ReferencePolicy;
-import org.apache.felix.scr.annotations.Service;
 import org.apache.sling.commons.osgi.Order;
```

Comment:
Let us try to get rid of all references to commons.osgi. Those are usually not necessary when relying on DS 1.4

Reply:
to be handled in https://issues.apache.org/jira/browse/SLING-10964